packages/ra-ui-materialui/src/auth/LoginForm.tsx

```diff
@@ -49,50 +49,44 @@ export const LoginForm = (props: LoginFormProps) => {
     };
 
     return (
-        <Form
-            onSubmit={submit}
-            mode="onChange"
-            render={({ handleSubmit }) => (
-                <Root onSubmit={handleSubmit} noValidate className={className}>
-                    <CardContent className={LoginFormClasses.content}>
-                        <TextInput
-                            autoFocus
-                            source="username"
-                            label={translate('ra.auth.username')}
-                            validate={required()}
-                            fullWidth
-                        />
-                        <TextInput
-                            source="password"
-                            label={translate('ra.auth.password')}
-                            type="password"
-                            autoComplete="current-password"
-                            validate={required()}
-                            fullWidth
-                        />
+        <Form onSubmit={submit} mode="onChange" noValidate>
```

Comment:
You still have to make `Root` a styled version of `Form` to allow children styling